appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -202,7 +214,7 @@ public void resetMetricsSecureProperty() {
         MetricRegistry registry = getRegistry(registryName);
         Map<String, Metadata> metricMetadataMap = registry.getMetadata();
         if (metricMetadataMap.containsKey(metricName)) {
-            return singletonMap(metricName, metricMetadataMap.get(metricName));
+            return Collections.singletonMap(metricName, metricMetadataMap.get(metricName));
```

Comment:
Does "Collections.singletonMap" really make the code more readable than "singletonMap"?

Reply:
Yes.

Reply:
What does it add for semantics knowing that the method comes from the `Collections ` class?

What you need to know reading the code is that a `singletonMap` is returned. Is there also another type of `singletonMap` that is not a `Collection`?

Reply:
By just calling `singletonMap()` I would presume that is was directly present in the class or a superclass, it is non-obvious that it comes from the `Collections` class.

Reply:
If it's only about "where it comes from" and not "the meaning itself isn't clear", then it doesn't matter ;)

Reply:
I would still say it isn't clear is a local method could well do data processing, which the one in the Collections framework does not.